packages/nimble/inst/CppCode/GNUmakefile

```diff
@@ -8,7 +8,12 @@ ifndef R_SHARE_DIR
 endif
 
 # Don't have to succeed in including this if we are calling cleanup in the nimble package directory.
--include $(R_HOME)/etc$(R_ARCH)/Makeconf
+# Following three lines result in much smaller libnimble.a or equivalent by removing -g flags
```

Comment:
The human-readable directions were intended more for ad-hoc testing on different operating systems.  But sure, turning debugging flags back on optionally could be useful.  I think we can add an option through the `configure.ac` to take an option to `R CMD install ...`.

Right, *two* lines.

Two additional changes I see: 

1. add `--vanilla` to `Rscript` call.

2. Add failsafe logic that checks if `Makeconf` exists locally and if not (meaning the `Rscript` job failed), use existing behavior of including Makeconf from R installation location.